twitchobserver/twitchobserver.py

```diff
@@ -242,7 +241,7 @@ def stop(self, force_stop=False):
 
         # Wait until all outbound events are sent
         while self._outbound_event_queue and not force_stop:
-            time.sleep(self._outbound_send_rate)
+            time.sleep(1 / self._outbound_sending_rate)
```

Comment:
Again, why are you inverting the rates here? The calculation of the poll rate should be correct.